configs/regnet/README.md

```diff
@@ -0,0 +1,37 @@
+# Designing Network Design Spaces
+
+## Introduction
+
+[ALGORITHM]
```

Comment:
Currently we use a commented tag. So we should change [ALGORITHM] to  <!-- [ALGORITHM] -->.
See https://github.com/open-mmlab/mmclassification/pull/226 for your reference.